### PR TITLE
Add a fashion MNIST example that can run on TPUs.

### PR DESCRIPTION
**Pull request by akshaym**
@iganichev 

Hey Igor, please take a quick look.

Thanks!

### CODE REVIEW

**Comment by lamberta** [comment]
Colab stage: https://colab.research.google.com/github/akshaym/docs/blob/master/site/en/tutorials/eager/_fashion_mnist_tpu.ipynb

**Comment by akshaym** [comment]
Hey @lamberta, @MarkDaoust,

Gentle ping on this. Do you have any comments about this PR?

Thanks!

**Comment by lamberta** [comment]
Thanks, Akshay. I think we're going to create a separate collection for these TPU examples since we're getting a few now. We can take it over and add all the formatting and stuff.
But just so I'm clear, the headline/purpose here is: "Eager execution on TPUs", yes?

**Comment by akshaym** [comment]
Yes, that would be the most appropriate title.

File: site/en/tutorials/eager/_fashion_mnist_tpu.ipynb [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+        "          tf.contrib.cluster_resolver.TPUClusterResolver(tpu='grpc://' + os.environ['COLAB_TPU_ADDR'])\n",
+        "      )\n",
+        "  )\n",
+        "  tpu_model.compile(\n",
+        "      optimizer=tf.train.AdamOptimizer(learning_rate=1e-3, ),\n",
+        "      loss=tf.keras.losses.sparse_categorical_crossentropy,\n",
+        "      metrics=['sparse_categorical_accuracy']\n",
+        "  )\n",
+        "\n",
+        "  def train_gen(batch_size):\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/eager/_fashion_mnist_tpu.ipynb]: Is there no way to use `tf.data` here?

**Comment by MarkDaoust** [comment]
We can work with this.

The only part I won't know how to describe is why we can't use `tf.data` to train on the full TPU.

Do any of you have some insight into why that is?

**Comment by lamberta** [comment]
Thanks, Akshay. I'm going to close this PR since this has changed a bit.

The https://www.tensorflow.org/tfrc page has been updated to feature more official TPU examples. It also links to a [FAshion MNIST with Keras and TPUs](https://colab.research.google.com/github/tensorflow/tpu/blob/master/tools/colab/fashion_mnist.ipynb) notebook.

These notebook examples live in the [tensorflow/tpu](https://github.com/tensorflow/tpu/tree/master/tools/colab) repo and I'd suggest adding TPU examples there if this is still needed. If, for some reason, this doesn't fit in that repo, then we have a new [tensorflow/examples](https://github.com/tensorflow/examples) repo that we can add this to. Thanks

**Pull request closed without merging**